addOns/oast/CHANGELOG.md

```diff
@@ -6,7 +6,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
-
+### Added 
+-An option to insert payloads through a context menu for all services, including Boast, Callback, and Interactsh, in locations where editing/sending HTTP/WebSocket messages is permitted (e.g. Manual Request Editor, Break, Fuzzer).  
```

Comment:
The context menu is shown in all text components not just the ones mentioned here.

Reply:
ideally should it be shown to all the text components or just the ones mentioned ?

Reply:
I thinks it's good to allow to paste in more places (e.g. one can paste into the Encoder dialogue to craft an encoded payload more easily).

Reply:
will it be fine if I allow it in all the text components ?

Reply:
There will be places where it does not make sense to paste a payload but given there will be more places where it's useful to allow to paste I think it's fine to apply to all for now (we can always change that later if really needed).

Reply:
okk